mappings/net/minecraft/client/model/Model.mapping

```diff
@@ -4,11 +4,13 @@ CLASS net/minecraft/class_3879 net/minecraft/client/model/Model
 	FIELD field_21343 layerFactory Ljava/util/function/Function;
 	METHOD <init> (Ljava/util/function/Function;)V
 		ARG 1 layerFactory
+	METHOD accept (Ljava/lang/Object;)V
+		ARG 1 part
 	METHOD method_23500 getLayer (Lnet/minecraft/class_2960;)Lnet/minecraft/class_1921;
 		ARG 1 texture
 	METHOD method_2828 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;IIFFFF)V
 		ARG 1 matrices
```

Comment:
Would it make sense for the `matrices` parameters to instead be like `transform`?

Reply:
Maybe? The standard we (fabric) seems to have settled on is `matrices`. Maybe such a change can come in conjunction with a rename of the class.